src/map.cpp

```diff
@@ -6742,8 +6742,14 @@ void map::loadn( const tripoint &grid, const bool update_vehicles )
 template <typename Container>
 void map::remove_rotten_items( Container &items, const tripoint &pnt )
 {
+    temperature_flag flag;
+    if( ter( pnt ) == t_rootcellar ) {
```

Comment:
Where are the fridges checked?

Reply:
Fridges are vehicle parts.
Items inside vehicle containers are not touched here.